spec/requests/api/articles_show_request_spec.rb

```diff
@@ -21,6 +25,10 @@
       it ':body' do
         expect(response_json['article']).to have_key 'body'
       end
+
+      it ':premium' do
```

Comment:
update this test to check if premium is false and that u can see more then 100 characters

Reply:
done